gdnative-bindings/build.rs

```diff
@@ -9,30 +9,18 @@ use std::process::Command;
 fn main() {
     let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
 
-    let bindings_types_rs = out_path.join("bindings_types.rs");
-    let bindings_traits_rs = out_path.join("bindings_traits.rs");
-    let bindings_methods_rs = out_path.join("bindings_methods.rs");
+    let generated_rs = out_path.join("generated.rs");
 
-    let mut types_output = BufWriter::new(File::create(&bindings_types_rs).unwrap());
-    let mut traits_output = BufWriter::new(File::create(&bindings_traits_rs).unwrap());
-    let mut methods_output = BufWriter::new(File::create(&bindings_methods_rs).unwrap());
+    {
+        let mut output = BufWriter::new(File::create(&generated_rs).unwrap());
 
-    // gdnative-core already implements all dependencies of Object
-    let to_ignore = { strongly_connected_components(&Api::new(), "Object", None) };
+        // gdnative-core already implements all dependencies of Object
+        let to_ignore = { strongly_connected_components(&Api::new(), "Object", None) };
 
-    generate_bindings(
-        &mut types_output,
-        &mut traits_output,
-        &mut methods_output,
-        Some(to_ignore),
-    )
-    .unwrap();
-
-    drop(types_output);
-    drop(traits_output);
-    drop(methods_output);
+        generate_bindings(&mut output, Some(to_ignore)).unwrap();
+    }
 
-    for file in &[bindings_types_rs, bindings_traits_rs, bindings_methods_rs] {
+    for file in &[generated_rs] {
```

Comment:
I think the `file` variable can just be renamed to `generated_rs`, the for loop only executes once :)